src/set_theory/cardinal_ordinal.lean

```diff
@@ -358,6 +358,9 @@ max_le
   (by simpa only [one_mul] using
     mul_le_mul_right' (one_lt_omega.le.trans ha) b)
 
+@[simp] theorem mul_mk_eq_max {α β : Type*} [infinite α] [infinite β] : #α * #β = max (#α) (#β) :=
```

Comment:
Is `max` the simp normal form? It's fine with me, but I could imagine that there's not a clear winner for either side of this equation.

Reply:
It is the simp normal form. The reason I want this is that it makes it possible to reduce `#α × #α` to `#α`.